xcodeproj/internal/opts.bzl

```diff
@@ -221,7 +221,8 @@ def _process_base_compiler_opts(
         if skip_next:
             skip_next -= 1
             continue
-        if "__BAZEL_XCODE_" in opt:
+        if (opt.startswith("-F__BAZEL_XCODE_") or
+            opt.startswith("-I__BAZEL_XCODE_")):
```

Comment:
@keith There's no harm if these are set twice, right? I should probably just not filter it in case someone does something additional.